api/src/main/java/org/eclipse/microprofile/config/spi/ConfigBuilder.java

```diff
@@ -42,77 +42,83 @@
 @org.osgi.annotation.versioning.ProviderType
 public interface ConfigBuilder {
     /**
-     * Add the default config sources appearing on the builder's classpath
-     * including:
-     * <ol>
-     * <li>System properties</li>
-     * <li>Environment properties</li>
-     * <li>/META-INF/microprofile-config.properties</li>
-     * </ol>
+     * Add the <a href="ConfigSource.html#default_config_sources"><em>default configuration sources</em></a>
+     * to the configuration being built.
      *
-     * @return the ConfigBuilder with the default config sources
+     * @return this configuration builder instance
      */
     ConfigBuilder addDefaultSources();
 
     /**
-     * Add the config sources appearing to be loaded via service loader pattern
+     * Add the all configuration sources which can be <a href="ConfigSource.html#discovery">discovered</a> from
```

Comment:
The reference needs to be the concrete one. For refering its own spec page, it is tricky as the link is not available as yet.

Reply:
This doesn't refer to a spec page, it creates a link to another JavaDoc element. It even works in IDEs (well IntelliJ at least).

Reply:
ack.